src/app/chat/event_handling.nim

```diff
@@ -37,7 +37,7 @@ proc handleChatEvents(self: ChatController) =
     self.view.hideLoadingIndicator()
     self.view.updateUsernames(evArgs.contacts)
     self.view.updateChats(evArgs.chats)
-    self.view.pushMessages(evArgs.messages)
+    self.view.pushMessages(evArgs.messages, evArgs.chats)
```

Comment:
Maybe, we should use another API than pushMessages to update the member list?
What about pushMembers?